docs/usage/secrets_engines/totp.rst

```diff
@@ -0,0 +1,91 @@
+TOTP
+====
+
+Create Key
+-------------------
+
+:py:meth:`hvac.api.secrets_engines.totp.create_key`
+
+.. code:: python
+
+	import hvac
+	client = hvac.Client()
+
+        create_key_response = client.secrets.totp.create_key(
+           'mykey',
+           url="otpauth://totp/Google:test@gmail.com?secret=Y64VEVMBTSXCYIWRSHRNDZW62MPGVU2G&issuer=Google"
+        )
+        print('New TOTP key response: {}'.format(create_key_response))
```

Comment:
nitpick: To simplify the docs a little bit, do you mind using f-strings in your examples? Since the next release will be 3.6 and greater, I'd like for HVAC to start using modern Python as much as possible.
```
print(f'New TOTP key response: {create_key_response}')
```